templates/terraform/encoders/backend_service.go.erb

```diff
@@ -43,8 +43,12 @@ for _, backendRaw := range backends {
 	if !ok {
 		continue
 	}
-	if strings.Contains(backendGroup.(string), "global/networkEndpointGroups") {
-		// Remove `max_utilization` from any backend that belongs to a global NEG. This field
+	match, err := regexp.MatchString("projects/(?P<project>[^/]+)/regions/(?P<region>[^/]+)/networkEndpointGroups/(?P<name>[^/]+)", backendGroup.(string))
+	if err != nil {
+		return nil, err
+	}
+	if match || strings.Contains(backendGroup.(string), "global/networkEndpointGroups") {
```

Comment:
It feels a bit extra to me to have both regex matching and strings.contains. does it make sense to do just one or the other?

Reply:
Yeah, string.Contains worked for global NEGs as the key string `global/networkEndpointGroups` is contiguous, needed a regex to look for `regions/{{region}}/networkEndpointGroups`. I'll make them both use a regex